bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java

```diff
@@ -87,6 +87,50 @@ public Builder withScan(Scan scan) {
       return this;
     }
 
+    /*
+     * Overrides of the CloudBigtableConfiguration.Builder's with*() so that they return
+     * CloudBigtableScanConfiguration.Builder.
+     */
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Builder withProjectId(String projectId) {
+      return (Builder) super.withProjectId(projectId);
```

Comment:
@Override
public Builder withProjectId(String projectId) {
  super.withProjectId(projectId);
  return this;
}